src/accelerate/launchers.py

```diff
@@ -198,7 +217,25 @@ def train(*args):
                 launcher = PrepareForLaunch(function, distributed_type="MULTI_GPU")
                 print(f"Launching training on {num_processes} GPUs.")
                 try:
-                    start_processes(launcher, args=args, nprocs=num_processes, start_method="fork")
+                    rdzv_conf = rdzv_conf or {}
```

Comment:
Let's be explicit please rather than relying on python :) 
```suggestion
                    if rdvz_conf is None:
                        rdvz_conf = {}
```